src/Compilers/CSharp/Portable/Binder/LocalBinderFactory.cs

```diff
@@ -224,6 +224,11 @@ public override void VisitLocalFunctionStatement(LocalFunctionStatementSyntax no
                 var oldMethod = _containingMemberOrLambda;
                 _containingMemberOrLambda = match;
 
+                foreach (var parameter in node.ParameterList.Parameters)
+                {
+                    Visit(parameter.Default);
```

Comment:
>Visit(parameter.Default); [](start = 20, length = 25)

It looks like BindParameterDefaultValue would always crash without this code. This indicates that we probably do not have any tests that verify that default parameter values are handled properly for local functions. Please add some of the success tests that verify that it is possible to take advantage of the default values. Should also check if values make it to metadata.